ci/install_data_services.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status, or zero if all
+# commands in the pipeline exit successfully.
+set -eu
+
+# ###
+# 1. section: Define script variables.
```

Comment:
Please don't number sections, they will soon go stale. Also, with just one section that is stating the obvious, I don't think it makes sense to keep this comment at all.

Reply:
all removed for all *.sh